packages/components/src/components/tick-picker/tick-picker.tsx

```diff
@@ -8,6 +8,18 @@ import Text from '../text';
 
 const THROTTLE_INTERVAL_TIME = 100;
 
+type TTickPicker = {
+    default_value: string;
```

Comment:
default_value should be a number as per stories - packages/components/stories/tick-picker/README.md. can you double check the type for it?